Makefile

```diff
@@ -1,5 +1,5 @@
 UPDATE_URL=
-UPDATE_PUBKEY=
+UPDATE_PUBKEY=Md2rdsS+b6W0trgcqa5lAWP978Zj0sFmubJ252OPKwc=
```

Comment:
This is the public key that corresponds with the new signer key. They are used to in the end let `update.go` verify that this is a sane artifact.